### ISSUE TITLE
Support parsing Oracle TO_DATE() and TO_TIMESTAMP() functions

### ISSUE THREAD

**Issue by tehbilly** [jOOQ/jOOQ#7320]
### Expected behavior and actual behavior:

I am attempting to translate some flyway migrations into another database dialect, the majority seem to work well but some table alteration statements are causing problems.

### Steps to reproduce the problem:

Using the following query either with on the online translator or the `parseQuery(string)` call in code will return an error:

```sql
ALTER TABLE SOME_TABLE ADD SOME_DATE_COLUMN DATE DEFAULT TO_DATE('1970-01-01', 'yyyy-mm-dd') NOT NULL
```

### Versions:

- jOOQ: only / 3.10.5
- Java: 8
- Database (include vendor): n/a (but source: Oracle, target: Postgres)
- OS: Windows
- JDBC Driver (include name if inofficial driver): n/a

**Comment by lukaseder** [comment]
Thanks a lot for your report. Indeed, this parser feature is missing, despite the fact that we already support the function in the jOOQ API.

Will fix this early next week.

**Comment by lukaseder** [comment]
Fixed for jOOQ 3.11. Parser upgrade will be merged to 3.10.6 tomorrow, March 20, through #7171. The website will be updated tomorrow as well.

Thanks again for reporting!
